src/components/KModal/KModal.vue

```diff
@@ -299,7 +303,7 @@ onBeforeUnmount(async () => {
     justify-content: center;
     padding: var(--kui-space-70, $kui-space-70) var(--kui-space-50, $kui-space-50) var(--kui-space-0, $kui-space-0) var(--kui-space-50, $kui-space-50);
     position: fixed;
-    z-index: 1100;
+    z-index: v-bind(zIndex);
```

Comment:
```suggestion
    z-index: v-bind('zIndex');
```

issue: `v-bind` parameter needs to be wrapped by single quotes

Reply:
Yep, let's standardize by wrapping: https://vuejs.org/api/sfc-css-features.html#v-bind-in-css

Reply:
According to the docs seems like it isn't always required but I think we should always wrap since it's safer while still correct

Reply:
it's required when accessing properties of an object, etc.

In this particular case, referencing the prop should really be done via `v-bind('props.zIndex')` I believe? But yes quoted regardless